converter/complete.go

```diff
@@ -305,7 +305,19 @@ func Complete(src string, pos token.Pos, conf *Config) ([]string, int, int) {
 	return match, start, end
 }
 
+func removePrefixes(src, prefix string) string {
+	if strings.Index(src, prefix) >= 0 {
+		src = strings.Replace(src, prefix, "", 1)
+	}
+
+	return src
+}
+
 func complete(src string, pos token.Pos, conf *Config) ([]string, int, int) {
+	// Removing go and defer prefixes from the string; fixes #8
+	removePrefixes(src, "go ")
```

Comment:
These two function calls are no-op. You do not fix anything by these two lines.
`src = removePrefixes(removePrefixes(src, "go "), "defer ")`
will cause runtme panic `pos` is not changed properly.

Anyway, just removing "go " from src is not a right solution for #8 because `go` can be a prefix of an identifier (e.g. google) which users are typing now.

Reply:
Let me fix it in a proper way; I'll close this